### PR TITLE
Tony/core js sdk v1 page

### PR DESCRIPTION
**Pull request by tonyboylehub**

### CODE REVIEW

File: src/pages/core/blog/core-js-sdk-v1.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+
+#### validateBurn
+
+Returns a `boolean` value on whether the publicKey can burn the Asset.
+
+```ts
+export async function validateBurn(umi, { authority, asset, collection })
+```
+
+#### canUpdate()
```
Comment (danenbm) [inline src/pages/core/blog/core-js-sdk-v1.md:245]: Should be `validateUpdate()`

File: src/pages/core/blog/core-js-sdk-v1.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+  plugin: {
+    type: 'Attributes',
+    attributeList: [{ key: 'key', value: 'value' }],
+  },
+}).sendAndConfirm(umi)
+```
+
+### Lifecycle Wrapper now Requires Asset Objects
+
+The create/update/transfer/burn now require the full asset/collection objects in order to derive extra accounts if any.
```
Comment (danenbm) [inline src/pages/core/blog/core-js-sdk-v1.md:77]: Could you just add a phrase or sentence here saying when there would be extra accounts?

**Review by danenbm — CHANGES_REQUESTED** [review]

File: src/pages/core/blog/core-js-sdk-v1.md (hunk 2 above)
Comment (tonyboylehub) [reply src/pages/core/blog/core-js-sdk-v1.md]: "The create/update/transfer/burn now require the full asset/collection objects in order to derive extra accounts (such as external plugin adapter accounts) if any if any."

**Review by danenbm — APPROVED** [review]
LGTM!

**Pull request merged**